src/group_operations/multiplication_operation_abelian.jl

```diff
@@ -0,0 +1,396 @@
+end
+
+function inv_left_compose(
+    ::LieGroup{𝔽,AbelianMultiplicationGroupOperation},
+    g::AbstractArray{<:Any,0},
+    h::AbstractArray{<:Any,0},
+) where {𝔽}
+    return map(\, g, h)
+end
+
+function inv_left_compose!(
+    G::LieGroup{𝔽,<:AbelianMultiplicationGroupOperation}, k, g, h
+) where {𝔽}
+    return copyto!(k, inv_left_compose(G, g, h))
+end
+
+function inv_right_compose(
+    ::LieGroup{𝔽,AbelianMultiplicationGroupOperation}, g::Number, h::Number
+) where {𝔽}
+    return g * inv(h)
+end
+
+function inv_right_compose(
+    ::LieGroup{𝔽,AbelianMultiplicationGroupOperation},
+    g::AbstractArray{<:Any,0},
+    h::AbstractArray{<:Any,0},
+) where {𝔽}
+    return map(/, g, h)
+end
+
+function inv_right_compose!(
+    G::LieGroup{𝔽,<:AbelianMultiplicationGroupOperation}, k, g, h
+) where {𝔽}
+    return copyto!(k, inv_right_compose(G, g, h))
+end
+
+_doc_identity_element_abelian_mult = """
+    identity_element(G::LieGroup{𝔽,AbelianMultiplicationGroupOperation})
+    identity_element!(G::LieGroup{𝔽,AbelianMultiplicationGroupOperation}, e)
+
+Return the point representation of the [`Identity`](@ref),
+which for an [`AbelianMultiplicationGroupOperation`](@ref) is the one-element.
+
+This can be computed in `e` if `e` is `mutable`.
+"""
+
+@doc "$(_doc_identity_element_abelian_mult)"
+function identity_element(
+    ::LieGroup{𝔽,AbelianMultiplicationGroupOperation}, ::Type{T}
+) where {𝔽,T<:Union{Number,AbstractArray{0,<:Number}}}
+    return one(T)
+end
+function identity_element(
+    ::LieGroup{𝔽,AbelianMultiplicationGroupOperation}, ::Type{Array{T,0}}
+) where {𝔽,T<:Number}
+    return fill(one(T))
+end
+function identity_element(
+    ::LieGroup{𝔽,AbelianMultiplicationGroupOperation}, e::Number
+) where {𝔽}
+    return one(e)
+end
+
+@doc "$(_doc_identity_element_abelian_mult)"
+identity_element!(::LieGroup{𝔽,AbelianMultiplicationGroupOperation}, e) where {𝔽}
+
+function identity_element!(
+    ::LieGroup{𝔽,AbelianMultiplicationGroupOperation}, e::AbstractArray{<:Number,0}
+) where {𝔽}
+    return fill!(e, 1)
+end
+
+_doc_lie_bracket_abelmult = """
+    lie_bracket(::LieAlgebra{𝔽,AbelianMultiplicationGroupOperation}, X, Y)
+    lie_bracket!(::LieAlgebra{𝔽,AbelianMultiplicationGroupOperation}, Z, X, Y)
+
+Compute the Lie bracket ``[⋅,⋅]: $(_math(:𝔤))×$(_math(:𝔤)) → $(_math(:𝔤))``,
+which for the for the [`AbelianMultiplicationGroupOperation`](@ref) yields the zero vector of the [`LieAlgebra`](@ref) due to commutativity. 
+
+```math
+[X, Y] = XY-YX = 0
+```
+
+The computation can be done in-place of `Z` if `Z` is `mutable`.
+"""
+
+@doc "$(_doc_lie_bracket_abelmult)"
+function lie_bracket(
+    ::LieAlgebra{𝔽,O,<:LieGroup{𝔾,O}}, X::Number, Y::Number
+) where {𝔽,𝔾,O<:AbelianMultiplicationGroupOperation}
+    return zero(X)
+end
+
+@doc "$(_doc_lie_bracket_abelmult)"
+function lie_bracket!(
+    ::LieAlgebra{𝔽,O,<:LieGroup{𝔾,O}}, Z, X, Y
+) where {𝔽,𝔾,O<:AbelianMultiplicationGroupOperation}
+    return copyto!(Z, zero(Z))
```

Comment:
```suggestion
    return fill!(Z, 0)
```

Reply:
`fill!` should be generic enough

Reply:
Between those two, I can do whatever I want. I usually roll a dice and choose the one that is wrong ;)